src/styles/getMuiTheme.js

```diff
@@ -224,4 +226,12 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       borderColor: palette.borderColor,
     },
   }, muiTheme);
+
+  if (computedTheme.isRtl) {
```

Comment:
> I think some components rely on isRtl.

Yikes :scream_cat: , that point I did not realize. I thought the whole point of `isRtl` was so that components didn't have to be written in a way where they worried rtl vs ltr, and that it would be properly converted in prepareStyles.

Mm, I think we should consider removing the dependencies on `isRtl` from the components.

> This is a bit scary. :P What if they want to add transformation after rtl is applied?

In my view, if they wanted to do this, they would just simply supply their own list of transformers `[autoprefix, customTransformer, rtl]` and just set `isRtl` to false. In other words, `isRtl` is only for backwards compatibility purposes.

Now because of your other point regarding components dependent on `isRtl` that couldn't work.